eng/common/scripts/add-pullrequest-reviewers.ps1

```diff
@@ -0,0 +1,95 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $false)]
+    $GitHubUsers = "",
+
+    [Parameter(Mandatory = $false)]
+    $GitHubTeams = "",
+
+    [Parameter(Mandatory = $true)]
+    $PRNumber,
+  
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+)
+
+function CombineArrays($ExistingIds, $NewIds) {
+  if ($ExistingIds -and $ExistingIds.Length > 0) {
+    $results = $ExistingIds.Clone()
+  }
+  else {
+    $results = @()
+  }
+
+  foreach ($id in $NewIds) {
+    if (-not $results.Contains($id)) {
+      $results += $id
+    }
+  }
+
+  return ,$results
+}
+
+# at least one of these needs to be populated
+if (-not $GitHubUsers -and -not $GitHubTeams) {
+  Write-Host "No user provided for addition, exiting."
+  exit 0
+}
+
+$userAdditions = @($GitHubUsers.Split(",") | % { $_.Trim() } | ? { return $_ })
```

Comment:
What are you exactly trying to do with `? { return $_ }`? Are you trying to filter out $null entries? If so the pipeline will automatically do that so you don't need that where clause.

Reply:
Null and whitespace. Here's an example directly off pscore console.

![image](https://user-images.githubusercontent.com/45376673/87103965-639ec600-c20b-11ea-8fbe-95ef6eb7d371.png)

I'm not planning on adjusting this one.

Reply:
OK I think I understand what is happening it does remove whitespace because of the trim.

```
> $d = "a ,  b  s, s,a,,,,"
> $d -split "," | % { $_.Trim() } | ? { $_ }
a
b  s
s
a
```
I think wrapping it in the `@()` isn't necessary but I don't think it hurts either.